src/streamlink/plugins/tvplayer.py

```diff
@@ -60,40 +47,36 @@ def authenticate(self, username, password):
         res = self.session.http.get(self.login_url)
         match = self.login_token_re.search(res.text)
         token = match and match.group(1)
-        res2 = self.session.http.post(self.login_url, data=dict(email=username, password=password, token=token),
+        res2 = self.session.http.post(self.login_url, data=dict(email=username, password=password, _token=token),
                          allow_redirects=False)
```

Comment:
[PEP8: continuation line under-indented for visual indent](https://stackoverflow.com/a/15435837)

```suggestion
        res2 = self.session.http.post(
            self.login_url,
            data=dict(email=username, password=password, _token=token),
            allow_redirects=False
        )
```